src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -21,13 +29,13 @@ public boolean isFull() {
 		
 				protected int peekaboo() {
 					if (callCheck())
-						return -1;
+						return STACK_CAPACITY;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)